swap/src/config.rs

```diff
@@ -16,9 +16,23 @@ pub struct Config {
     pub monero_network: monero::Network,
 }
 
-impl Config {
-    pub fn mainnet() -> Self {
-        Self {
+// TODO: This trait is not needed
+pub trait GetConfig {
```

Comment:
Why are there traits for the config? 
Shouldn't it be enough to create static test configs that are passed in?

Reply:
Because I don't want to introduced `Fast/Slow` config swap release code.